client/components/rewind-credentials-form/index.jsx

```diff
@@ -274,7 +274,8 @@ export class RewindCredentialsForm extends Component {
 }
 
 const mapStateToProps = ( state, { siteId } ) => ( {
-	formIsSubmitting: isUpdatingJetpackCredentials( state, siteId ),
+	formIsSubmitting: 'pending' === getJetpackCredentialsUpdateStatus( state, siteId ),
+	requestStatus: getJetpackCredentialsUpdateStatus( state, siteId ),
```

Comment:
Was this line supposed to be here? We don’t seem to be using it and if we need it we can later change formIsSubmitting

Reply:
Yeah, `formIsSubmitting` is used as a boolean in 13 places throughout, it seems better to me to leave this as is instead of junk up the code with that same logic in 13 places.